OpenIPSL/UsersGuide/ReleaseNotes/v100.mo

```diff
@@ -5,7 +5,7 @@ class v100 "Version 1.0.0 (2016-12-16)"
 <p>First Release of OpenIPSL</p>
 <h4>What&apos;s Changed</h4>
 <ul>
-<li>Fixing the initialisation issue in OpenModelica for all of the generators by <a href=\"https://github.com/tinrabuzin\">@tinrabuzin</a> in <a href=\"https://github.com/OpenIPSL/OpenIPSL/pull/3\">#3</a></li>
+<li>Fixing the initialization issue in OpenModelica for all of the generators by <a href=\"https://github.com/tinrabuzin\">@tinrabuzin</a> in <a href=\"https://github.com/OpenIPSL/OpenIPSL/pull/3\">#3</a></li>
```

Comment:
I don't know about changing the spelling from British english to American would be necessary, since those changes were done by Tin. @dietmarw what do you think?

Reply:
Well if it was for me I would make it BE all the way but I might be alone with that.

Reply:
haha yeah, but considering that we have both American and British speakers here, I would not consider this a typo, so I would not change it. Do you prefer uniform usage or a an approach in which American and British spellings are both considered?

Reply:
In this case, I think I'm in favour of the freedom of personal flavour. So since the @tbeu as PR provider would like to use AE I leave that in but will remove any BE->AE rewrite from this PR.